src/UbiUdp.cpp

```diff
@@ -104,3 +104,8 @@ IPAddress UbiUDP::getServerIp() {
 
   return IPAddress(169, 55, 61, 243);
 }
+
+tcpMap UbiUDP::getMultipleValues(const char *deviceLabel, const char *variableLabels) {
```

Comment:
Do not understand why are you setting 2 const char* parameter to this function if they are not being used.